### ISSUE TITLE
False positives when using SafeTestsets.jl

### ISSUE THREAD

**Issue by densmojd** [julia-vscode/StaticLint.jl#401]
I've been using  `SafeTestset.jl` for testing, i.e., my `runtests.jl` file looks like
```julia
@safetestset "test1" include("test1.jl")
@safetestset "test2" include("test2.jl")
```
etc.  `test1.jl` defines a variable, and `test2.jl` defines a function with the same name.  This generates the message "Cannot define a function; it already has a value."  However, `@safetestset` places the code in `test1.jl` and `test2.jl` in seperate modules, so there shouldn't be a name collision.
